src/docs/fund.yaml

```diff
@@ -0,0 +1,376 @@
+tags:
+  - name: Fund
+    description: API to manage funds.
+paths:
+  /api/fund:
+    post:
+      tags:
+        - Fund
+      summary: create a new fund
+      description: create a new fund
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/newFund"
+      responses:
+        201:
+          description: Authenticated user succesfully created fund.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Fund"
+        401:
+          description: user not sigend in
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: boolean
+                    example: true
+                  message:
+                    type: string
+                    example: "Invalid Token: No authorization token was found"
+        422:
+          description: This status is returned when the request is missing required fields or has an invalid values.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ValidationError"
+        500:
+          description: Internal server error
+  /api/fund/{id}/donate:
+    post:
+      tags:
+        - Fund
+      summary: Donate to a fund
+      description: Donates to a fund
+      operationId: donate
+      produces:
+        - application/json
+      consumes:
+        - application/json
+      parameters:
+        - in: path
+          name: id
+          description: The fund id
+          required: true
+          type: string
+        - in: body
+          name: body
+          required: true
+          schema:
+            properties:
+              amount:
+                type: number
+                description: The amount to donate
+                format: double
+                example: 10.00
+      responses:
+        201:
+          description: Successfully donated to fund
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Donation successful
+        422:
+          description: This status is returned when the request is missing required fields or has an invalid values.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ValidationError"
+        500:
+          description: Internal server error
+  /api/funds/:
```

Comment:
You seem to have created this route because you were not able to create a separate `/api/fund`, however, you could just use several verbs in the same endpoint like this: 
```
/api/fund
  get:
    ...
  post:
    ...
```